index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+    <link href="Inter-VariableFont_slnt,wght.ttf" rel="stylesheet">
+    <link href="CreteRound-Regular.ttf" rel="stylesheet">
+    <link href="Roboto-Regular.ttf" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+          <nav class="navigation">
```

Comment:
- [ ] I think it is better if you only wrap the hamburger in the ``nav`` tag because the ``nav`` tag is used for wrapping elements that navigate and the logo does not navigate the app 😉